src/baseline_bots/bots/__init__.py

```diff
@@ -1,10 +0,0 @@
-from baseline_bots.bots.baseline_bot import BaselineBot
```

Comment:
Why delete these?

Reply:
Why keep them?

Reply:
They allow people to do `from baseline_bots import BaselineBot`, etc., so you don't need to write the whole path when you want to import smtg.

Reply:
Okay, I'll put them back in